C/include/c4ConnectedClient.h

```diff
@@ -11,11 +11,29 @@
 //
 
 #pragma once
+#include "c4Base.h"
+#include "c4DocumentTypes.h"
 #include "c4ConnectedClientTypes.h"
 
 C4_ASSUME_NONNULL_BEGIN
 C4API_BEGIN_DECLS
 
+/** Result of a successful `c4client_getDoc` call. */
+typedef struct C4DocResponse {
+    C4HeapSlice docID;
+    C4HeapSlice revID;
+    C4HeapSlice body;
+    bool deleted;
+} C4DocResponse;
+
+/** Parameters describing a connected client, used when creating a C4ConnectedClient. */
+typedef struct C4ConnectedClientParameters {
+    C4Slice                             url;               ///<URL with database to connect
+    C4Slice                             options;           ///< Fleece-encoded dictionary of optional parameters.
+    const C4SocketFactory* C4NULLABLE   socketFactory;     ///< Custom C4SocketFactory
+    void* C4NULLABLE                    callbackContext;   ///< Value to be passed to the callbacks.
+} C4ConnectedClientParameters;
+
```

Comment:
These lines look like glitches from a merge. They were moved to C4ConnectedClientTypes.h yesterday.